lib/quadquizaminos/qna.ex

```diff
@@ -75,25 +75,20 @@ defmodule Quadquizaminos.QnA do
   end
 
   defp answers(content, answers) do
-    regex = ~r/# answers(?<answers>.*)#/iU
+    regex = ~r/# answers(?<answers>.*)#/iUs
 
-    case named_captures(regex, content) do
+    case Regex.named_captures(regex, content) do
       %{"answers" => answers} ->
-        answers
-        |> String.trim()
-        |> String.split(["-", "*"], trim: true)
-        |> Enum.with_index()
+        answers(answers)
 
       nil ->
-        nil
         answers(answers)
     end
   end
 
   defp answers(answers) do
     answers
-    |> String.replace("\n", "")
-    |> String.split(["-", "*"], trim: true)
+    |> String.split(["\n-", "\n*", "\n"], trim: true)
```

Comment:
Does a new line qualifily as an answer 🤔 , if you have a look at the split, the last item splits on a new line which means any content with new line is considered an answer. Did I get this right?

Reply:
No , new line doesn't qualify as an answer, in my case I have split on a new line, then trimmed it. I found it easier to remove new line this way , rather than  iterating through the list again to remove/replace the new line